jquery.fancytree/index.d.ts

```diff
@@ -21,18 +21,21 @@ interface JQuery {
 declare namespace Fancytree {
     interface Fancytree {
         $div: JQuery;
-
+        widget: any; //JQueryUI.Widget;
```

Comment:
Because JQueryUI.Widget is implemented in such a way that some fields like `element` and methods like `destroy` are not accessible.